src/main/java/com/jvm_bloggers/frontend/admin_area/AdminSocialChannelsPage.java

```diff
@@ -0,0 +1,49 @@
+package com.jvm_bloggers.frontend.admin_area;
+
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.form.Button;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.markup.html.form.TextArea;
+import org.apache.wicket.markup.html.form.TextField;
+import org.apache.wicket.model.CompoundPropertyModel;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wicketstuff.annotation.mount.MountPath;
+
+@MountPath("social-channels")
+public class AdminSocialChannelsPage extends AbstractAdminPage {
+
+    @SpringBean
+    private AdminSocialChannelsPageBackingBean backingBean;
+
+    private String link;
+    private String message;
+
+    public AdminSocialChannelsPage() {
+        initGui();
+    }
+
+    private void initGui() {
+        Form<AdminSocialChannelsPage>
+            facebookPostForm =
+            new Form<AdminSocialChannelsPage>("facebookPostForm",
+                new CompoundPropertyModel<AdminSocialChannelsPage>(this));
+        add(facebookPostForm);
+        Label linkLabel = new Label("linkLabel", "Link");
```

Comment:
These labels do not have to be present in Wicket. They could be only in html file without `wicket:id`.